ocsp_test.go

```diff
@@ -533,3 +533,21 @@ func TestInitOCSPCacheFileCreation(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestCacheServerDisabled(t *testing.T) {
+	_ = os.Setenv(cacheServerEnabledEnv, "false")
+
+	if _, err := buildSnowflakeConn(context.Background(), Config{
+		Account:  "testaccount",
+		User:     "testuser",
+		Password: "testpassword",
+		Host:     "testaccount.us-east-1.snowflakecomputing.com",
+	}); err != nil {
+		t.Error(err)
+	}
+
+	ocspURL := os.Getenv(cacheServerURLEnv)
+	if ocspURL != "" {
```

Comment:
I'm not sure if I understand it. According to the first line in the test, you just set an env variable and now you read another one. I believe this just reads env variable, there is no driver logic. The only reason why this could work is that `buildSnowflakeConn` removes this env var, which seems unlike. But if this should work, maybe we should enrich the test with setting `cacheServerUrlEnv` at the beginning of the test?